jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -240,39 +240,38 @@ public JibContainerBuilder addEnvironmentVariable(String name, String value) {
   }
 
   /**
-   * Sets the directories that the container will create as a volume. Volumes allows persistent
-   * data.
+   * Sets the directories that may hold externally mounted volumes.
    *
    * <p>This is similar to <a href="https://docs.docker.com/engine/reference/builder/#volume">{@code
    * VOLUME} in Dockerfiles</a>.
    *
-   * @param volumes the list of volumes
+   * @param volumes the list of directories to set as volumes
```

Comment:
Actually, how about changing "directories" to "directory paths on the container filesystem"

Reply:
That's sounds better indeed.